public/src/pages/auth/Login/Login.tsx

```diff
@@ -124,7 +124,7 @@ export const Login: FC = () => {
     getOidcClient().then((client) => setOidcClient(client));
   };
 
-  const extractLogoBgColor = (): string | null => {
+  const extractLogoBgColor = (): string | undefined => {
```

Comment:
@derevnjuk Since the return type of `searchParams.get('logobgcolor')` is `string | null`, this change breaks the build with a type error. Either this function's return type should be `string | null` or `return searchParams.get('logobgcolor');` should be changed to `return searchParams.get('logobgcolor') || undefined;`

Reply:
@tamirGer you want to say there was a compile-time error before? Seems it is time to set up CI to avoid similar occurrences in the future.